src/test/developTest/resources/wdr/wdr_sendung_day.html

```diff
@@ -0,0 +1,1334 @@
+<div class="menueToggle hasArrow" data-ctrl-navprogram-nav1-trigger="{'action':{'default':['closeOtherEntries','toggleBody']}}" style="position: static; cursor: default;">
+<a href="#">Unterpunkte auf-/zuklappen</a>
+</div>
+<ul class="menu" data-ctrl-navprogram-nav1-body="{}">
+<li >
+<span class="collapsed subressort" >
+<a href="/mediathek/video/online-first/index.html">Vorab ansehen</a>
+</span>
+</li>
+<li >
+<span class="collapsed subressort" >
+<a href="/mediathek/video/live/index.html">Livestream</a>
+</span>
+</li>
+<li >
+<span class="collapsed subressort active" >
+<a href="/mediathek/video/sendungverpasst/index.html">Sendung verpasst</a>
+</span>
+</li>
+<li >
+<span class="collapsed subressort" >
+<a href="/mediathek/video/sendungen-a-z/index.html">Sendungen A-Z</a>
+</span>
+</li>
+</ul>
+<div class="clearMe"></div>
+</div>
+</div>
+</li>
+<li class="audio">
+<a href="/mediathek/audio/index.html">
+<span>Audios</span>
+</a>
+</li>
+</ul>
+</div>
+</div>
+</div>
+<!-- wdr -->
+
+<a name="goToContent" class="hidden"></a>
+
+<div class="section sectionA">
+<div class="con clean">
+<div class="modCon">
+<div class="mod modA modHeadline seo">
+<div class="box">
+<h1 class="headline">Rückschau chronologisch</h1>
+</div>
+</div>
+</div>
+</div>
+</div>
+<noscript>
+<div class="section sectionA">
+<div class="con">
+<div class="modCon">
+<div class="mod modA modStage">
+<div class="boxCon">
+<div class="box">
+<div class="teaser">
+<div class="mediaCon">
+<div class="media mediaA">
+<img alt="Javascript-Fehler" title="Javascript-Fehler" src="/resources/img/wdr/various/error.jpg" class="img" />
+</div>
+</div>
+<div class="textWrapper">
+<h4 class="headline">Diese Seite ben&ouml;tigt JavaScript. Bitte &auml;ndern Sie die Konfiguration Ihres Browsers.</h4>
+<p class="teasertext">Falls JavaScript in Ihrem Browser deaktiviert wurde, &auml;ndern Sie dies bitte &uuml;ber das
+entsprechende Einstellungs-Men&uuml; Ihres Browsers. Nur mit aktiviertem JavaScript kann unser Angebot richtig dargestellt
+und mit allen Funktionen genutzt werden.</p>
+</div>
+<!-- googleon: all -->
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</noscript>
+<div class="section sectionC" data-ctrl-sectionlayout-entry="{}">
+<div class="con">
+<div class="modCon">
+<div class="mod modA">
+<div class="boxCon">
+<div class="box modCal">
+<div class="form">
+<div id="cal-r0" class="calendar unitA" data-ctrl-calendar="{}">
+<h4 class="hidden">Sendungskalender von </h4>
+<ul class="clearfix">
+<li class="prev">
+<a href="/mediathek/video/sendungverpasst/sendung-verpasst-100~_mon-092017_tag-03102017.html" rel="nofollow">
+<span class="hidden">Vorheriger Monat</span>
+</a>
+</li>
+<li class="next">
+<a href="/mediathek/video/sendungverpasst/sendung-verpasst-100~_mon-112017.html" rel="nofollow">
+<span class="hidden">N&auml;chster Monat</span>
+</a>
+</li>
+</ul>
+<table summary="Kalender: Oktober 2017">
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "summary" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AS1827)